crates/bevy_render/src/entity.rs

```diff
@@ -106,7 +110,11 @@ impl OrthographicCameraBundle {
                 name: Some(base::camera::CAMERA_3D.to_string()),
                 ..Default::default()
             },
-            orthographic_projection: Default::default(),
+            orthographic_projection: OrthographicProjection {
+                scaling_mode: ScalingMode::FixedVertical,
```

Comment:
For 3D applications, I think `ScalingMode::FixedVertical` makes more sense as the default, because it behaves similarly to the perspective projection (does not reveal extra content as you resize the window). This seems to me like the expected behavior for 3D.